nano/secure/ledger.cpp

```diff
@@ -34,13 +34,12 @@ class rollback_visitor : public nano::block_visitor
 		}
 		if (!error)
 		{
-			nano::account_info info;
-			[[maybe_unused]] auto error (ledger.store.account.get (transaction, pending.source, info));
-			debug_assert (!error);
+			auto info = ledger.account_info (transaction, pending.source);
+			debug_assert (info);
```

Comment:
I'm wondering whether those asserts inside `rollback_visitor` should become release asserts. The check itself is very cheap